Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -80,6 +80,8 @@ public void Initialize()
 
             m_Cfg = Configuration.GetDefaultConfiguration();
             RegisterActions(m_Cfg);
+
```

Comment:
InputSystem.actions and InputSystem.onActionsChange are only available when UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS is defined, but there are no symbolic checks here. In addition this is compiled into its own assembly so cannot detect symbols from Input System defines in asmdef and this seems to only rely on UNITY version which makes this broken if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS is undefined?

Reply:
True, but that issue is not introduced by this PR.